Aquality.Selenium.Core/src/Aquality.Selenium.Core/Utilities/FileReader.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace Aquality.Selenium.Core.Utilities
+{
+    /// <summary>
+    /// Utility methods to read files.
+    /// </summary>
+    public static class FileReader
```

Comment:
maybe better to make it private/internal?

Reply:
no, we use it e.g. to read js from embedded resources in Web library